source/includes/extracts-atlas-search-facts.yaml

```diff
@@ -0,0 +1,24 @@
+ref: fact-text-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. A legacy text search capability is available for users 
+  self-managing MongoDB deployments. 
+
+---
+ref: fact-regex-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
```

Comment:
[suggestion]

I'd suggest only having the linked text be "MongoDB Atlas Search", rather than including "premium full-text search solution" in the link. I think reducing the amount of blue text will make this a bit more readable.